source/containers.tex

```diff
@@ -15328,16 +15328,14 @@
 \begin{itemdescr}
 \pnum
 \constraints
+Let \tcode{S} be \tcode{sizeof...(OtherSizeTypes)}.
```

Comment:
Hm, elsewhere we use `S` to denote an "index space", but I suppose there's enough context here to not make that confusing.

Reply:
@jensmaurer Mind if we use `R` here?

Reply:
And https://github.com/cplusplus/draft/pull/5635/files#diff-3b5851f7056b7c68cb3ba2ab51cf0b75a780c6a11e5e181770067700454ace7bR17131, for consistency.

Reply:
Yes, thanks.

I suppose we could also use `N` instead of `R`, then it'd be the same as the template parameter... thoughts?

Reply:
Then why not `R`? `Rank` could work, as well.

Reply:
I am currently using `R` for the local expression alias, yes, but I was suggesting that I could use `N` instead, since we also have another constructor with a template parameter `N` which serves the same role in the constraints there as the `R` does here.

Reply:
That makes sense. `N` seems better.

Reply:
Sounds fine. `R` is often a `range`, so we should avoid the letter if possible.

Reply:
OK, `N` it is then!